shell/gpu/gpu_surface_gl.cc

```diff
@@ -37,29 +37,34 @@ bool GPUSurfaceGL::Setup() {
     return false;
   }
 
-  if (!delegate_->GLContextMakeCurrent()) {
-    // Could not make the context current to create the native interface.
-    return false;
-  }
+  context_ = delegate_->GetGrContext();
+  srgb_support_ = delegate_->SupportsSRGB();
+
+  if (!context_) {
```

Comment:
If you cannot obtain the GrContext, you should probably return `false` early here. You can going to do a null pointer dereference on the `context_->setResourceCacheLimits` call later.